File: Insomnia.Core/ClientApp/src/app/raider/raider.ts
import { DkpItem } from "./dkp-item";

export class Raider {
  characterClass: string;
  name: string;
  dkp: number;
  dkpItems: DkpItem[];

  constructor(characterClass: string, name: string, dkp: number, dkpItems: DkpItem[]) {
    this.characterClass = characterClass;
    this.name = name;
    this.dkp = dkp;
    this.dkpItems = dkpItems;
  }
}

File: Insomnia.Core/Models/Raider.cs
namespace Insomnia.Core.Models
{
    public class Raider
    {
        public string Name { get; set; }
        public string CharacterClass { get; set; }
        public double Dkp { get; set; }
        public DkpItem[] DkpItems { get; set; }
    }
}
